### DISCUSSION TITLE
Welcome to oculante Discussions!

### DISCUSSION THREAD

**Discussion by woelper** [woelper/oculante#75] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 0323pin** [comment]
Ok, let's get this rolling 😄 

I'm a pkgsrc (NetBSD) maintainer/dev that likes `rust` and discovering new tools. `oculante` is a nice project for when you don't need all the _bling_ from GIMP and just want to view images and do some edits on a really nice environment.

Things I use on a daily basis on my NetBSD system:
WM: LeftWM
Terminal: Alacritty
Shell: elvish
Editor(s): vim and featherpad
Other: dua-cli, wiki-tui, lsd, tuifeed and spotify-player.

**Comment by woelper** [comment]
Yay! Welcome!

I am an Open Source & Linux fan, and started with OpenSuse, RedHat and Irix long ago.
I used to run Gentoo on all linux machines I owned, but switched to PopOS to keep fans from spinning and my time going into package resolution :) 
I use WezTerm, VsCode (Trying Lapce too) and a bunch of other random stuff, mostly on a Tuxedo Pulse 15 laptop.

Really happy to have some people using this image viewer, and excited about feedback and requests, since that is what makes it better!
